java/src/jmri/jmrix/AbstractSerialPortController.java

```diff
@@ -236,25 +236,29 @@ public String getCurrentBaudRate() {
         return mBaudRate;
     }
 
+    /**
+     * {@inheritDoc}
+     */
     @Override
     public String getCurrentBaudNumber() {
```

Comment:
@bobjacobsen Following changes in this file (up to line 261) could be added to fix 4.16.

Reply:
This seems to only change the Javadoc on the `getCurrentBaudNumber()` method, it doesn't have any functional changes.

Reply:
I meant to point to the changes to the method.